docs/src/main/asciidoc/security-authorize-web-endpoints-reference.adoc

```diff
@@ -65,15 +65,41 @@ quarkus.http.auth.permission.roles1.policy=role-policy1
 ----
 <1> This permission references the default built-in `permit` policy to allow `GET` methods to `/public`.
 In this case, the demonstrated setting would not affect this example because this request is allowed anyway.
-<2> This permission references the built-in `deny` policy for `/forbidden`.
+<2> This permission references the built-in `deny` policy for both `/forbidden` and `/forbidden/` paths.
 It is an exact path match because it does not end with `*`.
 <3> This permission set references the previously defined policy.
 `roles1` is an example name; you can call the permission sets whatever you want.
 
-[WARNING]
+[IMPORTANT]
 ====
-The exact path `/forbidden` in the example will not secure the `/forbidden/` path.
-It is necessary to add a new exact path for the `/forbidden/` path to ensure proper security coverage.
+The exact path `/forbidden` in the example above also secures the `/forbidden/` path.
+This way, the `forbidden` endpoint in the example below is secured by the `deny1` permission.
+
+[source,java]
+----
+package org.acme.crud;
+
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+
+@Path("/forbidden")
+public class ForbiddenResource {
+    @GET
+    public String forbidden() { <1>
+        return "No!";
+    }
+}
+----
+<1> Both `/forbidden` and `/forbidden/` paths needs to be secured in order to secure the `forbidden` endpoint.
+
+If you need to secure only the `/forbidden` path, please add the exclamation mark at the end of the exact path pattern like in the example below:
+
+[source,properties]
+----
+quarkus.http.auth.permission.deny1.paths=/forbidden! <1>
```

Comment:
Hmm, it looks a bit technical. See, we optimize on having to list both `/a` and `a/` but then users would have to do `/a!` for this case...

Reply:
Yes, the point is how frequent is this case? Technically they are 2 different paths so we need to treat them this way (as these are HTTP permissions, not JAXRS permissions). I honestly don't know how often this is useful, but I'd be worried to change original behavior without compatibility option in migration guide.

Reply:
@michalvavrik Well, what if we say, if you want to differentiate between `/a` and `/a/` permissions then list `/a=permitted` instead of indirectly permitting it with `a/!`

Reply:
I.e, if we have `/a` and some other policy has `/a/` then we don't cover `/a/` with the `/a` permission and the same for `/a/`, if `/a` already exists somewhere else then we don't use the `/a/` permission to cover `/a`

Reply:
@michalvavrik Sorry, I meant to use a dedicate policy for this path, say if `/a/` is denied and the users want `/a` permitted then let them create a dedicated policy where `/a` is allowed, using a `permit` permission. Apologies for the confusion

Reply:
np @sberyozkin , that's pretty good idea. alright I agree with this, thank you!